hermes/src/state.rs

```diff
@@ -67,10 +81,27 @@ impl State {
             aggregate_state: RwLock::new(AggregateState::new(&mut metrics_registry)),
             benchmarks_endpoint,
             metrics_registry: RwLock::new(metrics_registry),
+            rpc_http_endpoint,
+            mapping_address,
+            price_feeds_metadata: RwLock::new(Default::default()),
         })
     }
 }
 
+pub async fn retrieve_price_feeds_metadata(state: &State) -> Result<Vec<PriceFeedMetadata>> {
+    let price_feeds_metadata = state.price_feeds_metadata.read().await;
+    Ok(price_feeds_metadata.clone())
+}
+
+pub async fn store_price_feeds_metadata(
+    state: &State,
+    price_feeds_metadata: &[PriceFeedMetadata],
+) -> Result<()> {
+    let mut price_feeds_metadata_write_guard = state.price_feeds_metadata.write().await;
+    *price_feeds_metadata_write_guard = price_feeds_metadata.to_vec();
+    Ok(())
+}
+
```

Comment:
I think it would be better to move them to `price_feeds_metadata` module (and potentially remove them)